packages/markdown-docx/src/rules.js

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
You should rename this file to `toOOXMLRules.js`. Since this package has two transformers - `OOXML -> CiceroMark` and `CiceroMark -> OOXML`, it will help remove the ambiguity.